packages/react/src/components/header/Header.stories.tsx

```diff
@@ -337,56 +342,181 @@ export const MinimalWithLocalization = (args) => {
-          fixedRightPosition
-          icon={<IconUser />}
-          id="action-bar-login"
-        >
-          <h3>Kirjautumisvalinnat</h3>
+        <Header.ActionBarItem label="Haku" icon={<IconSearch />} id="action-bar-search">
+          <Header.Search label="Hae palvelusta" onChange={searchChangeAction} onSubmit={searchSubmitAction} />
         </Header.ActionBarItem>
       </Header.ActionBar>
 
       <Header.NavigationMenu>
-        <Header.Link href="#" label="Sosiaali- ja terveyspalvelut" />
-        <Header.Link href="#" label="Kasvatus ja koulutus" />
-        <Header.Link href="#" label="Asuminen" />
+        <Header.Link
+          href="#"
+          label="Sosiaali- ja terveyspalvelut"
+          onClick={(event) => event.preventDefault()}
+          active
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Terveydenhoito"
+              active
+              dropdownLinks={[
+                <Header.Link href="#" label="Hammashoito" />,
+                <Header.Link href="#" label="Julkinen terveydenhoito" />,
+              ]}
+            />,
+            <Header.Link
+              href="#"
+              label="Senioripalvelut"
+              dropdownLinks={[
+                <Header.Link href="#" label="Viriketoiminta" />,
+                <Header.Link href="#" label="Kuntouttavat palvelut" />,
+              ]}
+            />,
+          ]}
+        />
+        <Header.Link
+          href="#"
+          label="Kasvatus ja koulutus"
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Kasvatus"
+              active
+              dropdownLinks={[
+                <Header.Link href="#" label="Varhaiskasvatus" />,
+                <Header.Link href="#" label="Esiopetus" />,
+              ]}
+            />,
+            <Header.Link
+              href="#"
+              label="Koulutus"
+              dropdownLinks={[
+                <Header.Link href="#" label="Perusopetus" />,
+                <Header.Link href="#" label="Toisen asteen koulutus" />,
+                <Header.Link href="#" label="Työväenopistot" />,
+              ]}
+            />,
+          ]}
+        />
+        <Header.Link
+          href="#"
+          label="Yritykset ja työ"
+          dropdownLinks={[
+            <Header.Link
+              href="#"
+              label="Työnantajat"
+              active
+              dropdownLinks={[<Header.Link href="#" label="Yritykset" />, <Header.Link href="#" label="Yrittäjät" />]}
+            />,
+            <Header.Link
+              href="#"
+              label="Työntekijät"
+              dropdownLinks={[<Header.Link href="#" label="Avoimet työpaikat" />]}
+            />,
+          ]}
+        />
       </Header.NavigationMenu>
     </Header>
-  );
+    <div id="content" />
+  </>
+);
+
+WithFullFeaturesCustomTheme.args = {
+  theme: {
+    '--color-focus-outline': 'var(--color-metro-dark)',
```

Comment:
This actually only works on SkipLink? Everything else seems to have the default bus focus colour.

Reply:
fixed